source/installation/modify-system-security.rst

```diff
@@ -3,17 +3,40 @@
 Modify System Security
 ======================
 
-#. Disable `SELinux`_ on the Open OnDemand host you intend on running the web
-   server on.
+.. _ood_selinux:
+
+SELinux
+--------
+
+.. DANGER::
+   Support for SELinux on the Open OnDemand host is currently considered an alpha feature.
+
+#. If you plan to use `SELinux`_ on the Open OnDemand host you must install the `ondemand-selinux` package.
+
+   .. code-block:: sh
+
+      sudo yum install ondemand-selinux
 
    .. note::
 
-      If SELinux is not disabled on the web server host machine this can lead
-      to permission errors when running the web applications. SELinux may also
-      need to be disabled on the web server if you want to take advantage of
-      certain software in your web applications such as being able to access a
-      Lustre file system.
+      OnDemand runs under the Apache `httpd_t` context.
+
+The OnDemand SELinux package makes several changes to allow OnDemand to run with SELinux enabled.
+
+* Set context of several ondemand-nginx directories and files.
+* Enable several booleans.
+* Apply a custom policy to allow some additional actions by processes in the `httpd_t` context.
+
```

Comment:
Maybe a link to the source in this block? I feel like if someone enables selinux they'll be interested in knowing what _exactly_ they're changing.

Also I get that a solid link here isn't available yet given we haven't pulled the other PR into master.

Reply:
Added links to where files will be once things are merged.